cypress/e2e/admin/forcePasswordChangeStudent.feature

```diff
@@ -0,0 +1,100 @@
+        When I enter '<role_to_manage>' email '<user_email>' in search input field
+        Then I can see the user with email '<user_email>' in the table
+
+        # admin generates the registeration link to proceed with student  registration
+        When I enter '<role_to_manage>' email '<user_email>' in search input field
+        When I click edit '<role_to_manage>' button for '<user_email>'
+        Then I click on the button Generate Personal Registration Link for student
+
+        # student perform registration via the generated link with PIN
+        Then I visit to the generated registration link
+        When I choose the language for the registration process
+        Then I click on the button Next to proceed to the age selection
+        When I select the age over 16 years for registration
+        Then I click on the button Next to proceed to the personal data information page
+        Then I see my first name '<user_firstname>'
+        Then I see my last name '<user_lastname>'
+        When I click on the button Next to proceed to the next step
+        When I accept the privacy and terms of use consents
+        Then I click on the button Next to proceed to the registration pin step
+        When I request a new registration pin
+        Then I retrieve the registration pin to enter it into the form for '<namespace>'
+        Then I click on the button Send and Get Started to successfully complete the registration process
+        Then I see the page user data summary
+
+        # newly registered student techer do the first login
+        When I visit the url for first login
+        When I enter the email assigned during user creation
+        When I enter the initial generated password
+        When I click on the button Login
+        Then I see the first login page section 1
+        When I click on the button Next in section 1
+        Then I see the section 2
+        When I click on the button Next in section 2
+        Then I see the section 3
+        Then I set a new password
+        Then I re enter the new password
+        When I click on the button Next to proceed
+        Then I click on the button Get started now in section 4
+        Then I see the dashboard
+
+        # admin changes the password for the student
+        Given I am logged in as a '<admin>' at '<namespace>'
+        When I click on administration in menu
+        When I go to '<role_to_manage>' administration
+        When I enter '<role_to_manage>' email '<user_email>' in search input field
+        When I click edit '<role_to_manage>' button for '<user_email>'
+        When I click on the button Change password
+        Then I see the pop-up window
+        Then I enter a new password in the pop-up window
+        Then I click on the button Save
+        Then I see the success message
+        Then I logout from the application
+
+        # Student do the login and set a new password
```

Comment:
make comments consistent some place you started with capital letters.

Reply:
done!